tfhe/docs/how_to/compress.md

```diff
@@ -40,8 +54,19 @@ fn main() {
 
     let cks = ClientKey::generate(config);
     let compressed_sks = CompressedServerKey::new(&cks);
+
+    println!(
+        "  compressed len: {}",
```

Comment:
The spaces are used to start printing compressed and decompressed sizes at the same column to more easily compare them (especially the difference in number of digits)

Actually it would be better to make them end at the same column. I'll try to do that instead

Reply:
I thinks its more common to have
```
compressed len:    1111
decompressed len: 1111

# or
Yes it would look better like 
```
compressed len    :  1111
decompressed len: 1111
```

rather than 
```
    compressed len:  1111
decompressed len: 1111
```

Reply:
Done